store/sqlstore/upgrade.go

```diff
@@ -777,8 +775,6 @@ func upgradeDatabaseToVersion5291(sqlStore *SqlStore) {
 
 func upgradeDatabaseToVersion530(sqlStore *SqlStore) {
 	if shouldPerformUpgrade(sqlStore, Version5291, Version5300) {
-		sqlStore.CreateColumnIfNotExistsNoDefault("FileInfo", "Content", "longtext", "text")
-
 		saveSchemaVersion(sqlStore, Version5300)
 	}
 }
```

Comment:
I am thinking there should be an additional `CreateColumnIfNotExistsNoDefault` call  for `RemoteId` to remove in here.

Reply:
Thanks, that one got overlooked 🤦